go.mod

```diff
@@ -7,9 +7,9 @@ require (
 	github.com/containerd/containerd v1.6.6
 	github.com/containernetworking/cni v1.1.1
 	github.com/containernetworking/plugins v1.1.1
-	github.com/containers/image/v5 v5.21.2-0.20220712113758-29aec5f7bbbf
+	github.com/containers/image/v5 v5.21.2-0.20220714132403-2bb3f3e44c5c
 	github.com/containers/ocicrypt v1.1.5
-	github.com/containers/storage v1.41.1-0.20220712184034-d26be7b27860
+	github.com/containers/storage v1.41.1-0.20220718173332-b10c469fda0a
```

Comment:
I'm wondering why we do not release new versions of those libraries to get the dependencies pinned and updated by dependabot? Is this intentional?